.github/CODEOWNERS

```diff
@@ -0,0 +1,5 @@
+# These owners will be the default owners for everything in
+# the repo. Unless a later match takes precedence,
+# @global-owner1 and @global-owner2 will be requested for
+# review when someone opens a pull request.
+*       @VladimirKravtsov @akirillov @rpalaznik @farhan5900 @samvantran @alembiewski
```

Comment:
why this should be a part of this PR? can you please exclude it and create a new one for this change?

Reply:
make sense

Reply:
#37